mathesar_ui/src/component-library/text-input/TextInput.svelte

```diff
@@ -21,6 +22,14 @@
 
   // Id for the input
   export let id: $$Props['id'] = undefined;
+
+  export let cssVariables: $$Props['cssVariables'] = undefined;
+  $: styleStringFromCssVariables = cssVariables
+    ? createStyleString(cssVariables)
+    : '';
+  $: style = $$restProps.style
+    ? styleStringFromCssVariables + $$restProps.style
+    : styleStringFromCssVariables;
```

Comment:
Can we have another common method for merging the style strings? Something like:
```
$: style = mergeStyleStrings(makeStyleStringFromCssVariables(cssVariables), $$restProps.style);
```

```
mergeStyleStrings: (args: ...(string|undefined)[]) => string
```

and move the logic to the common method.